ui/app/TokenList.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "Chaosnet supported tokens",
+  "tokens": [
+    {
+      "chainId": 58008,
```

This is the token on optimism
```suggestion
      "chainId": 10,
```